java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -316,6 +318,70 @@ public interface ServerManagementClusterCommands {
      */
     CompletableFuture<ClusterValue<Long>> lastsave(Route route);
 
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.<br>
+     * The command will be routed to all primary nodes.
+     *
+     * @see <a href="https://redis.io/commands/flushall/">redis.io</a> for details.
```

Comment:
Sho has left comments on some of my other PRs asking to update the links to valkey, can we do so in this PR as well?